### DISCUSSION TITLE
External links without redirection page?

### DISCUSSION THREAD

**Discussion by nilskamm** [pretix/pretix#2951] [Q&A] [answered]
Hi everyone!
We have some links in our ticketshop which will open sites from our own homepage in a new tab / window.
At the moment there is always the pretix redirection page coming up where you have to click, to get to our homepage.
Is there a way to deactivate that or mark some links as "safe" to not have the redirection page?
Thanks a lot everyone!

**Comment by raphaelm** [comment, marked as answer]
The intermediate page should only show up if the Referer header does not match the pretix domain. Please double check that the URL in pretix.cfg is correct and that your web server passes correct Referer and Host headers

**Comment by nilskamm** [comment]
Hey raphaelm,
thank you for your answer! 
If i have a problem in my configuration i don´t see / understand it. 
Mind to have a look on it?
The URL part of my pretix.cfg looks like this:

> [pretix]
instance_name=Lost Ideas Ticketshop
url=https://pretix.lost-ideas.com
currency=EUR
; DO NOT change the following value, it has to be set to the location of the
; directory *inside* the docker container
datadir=/data
trust_x_forwarded_for=on
trust_x_forwarded_proto=on
#loglevel=DEBUG, INFO, WARNING, ERROR, CRITICAL
loglevel=INFO


My location part in my nginx config:
>   location / {
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_set_header X-Forwarded-Proto $scheme;
    proxy_set_header X-Forwarded-Proto https;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header Host $host;
    proxy_pass http://pretix;
  }

Our pretix runs on https://tickets.lost-ideas.com and one link for example refers to https://www.lost-ideas.com/impressum

Thanks a lot!

**Reply by raphaelm** [reply]
The only issue I can recognize here is
```
proxy_set_header X-Forwarded-Proto $scheme;
proxy_set_header X-Forwarded-Proto https;
```
which seems redundant. Maybe that gets it confused? Otherwise it looks fine

**Reply by nilskamm** [reply]
I deleted the second line and now it is working fine! 
Thanks a lot again!
